plugin-it/pom.xml

```diff
@@ -27,7 +27,7 @@
                   </goals>
```

Comment:
Note to self: refactor integration test for wiremock extensions. The only thing we test here as of today is if `java.lang.ClassNotFoundException` is thrown or not. We don't know if the transformation is applied or not.

After merging this, I'll revert changes to this pom file and extend Jmeter JMX file to cater for 2 scenarios: one without transformations, and one with. We will need to check explicitly for what we expect to be transformed.

I'm happy to do that myself.